### DISCUSSION TITLE
Styles aren't applying, Invalid property error

### DISCUSSION THREAD

**Discussion by Edgardo813** [chakra-ui/chakra-ui#6468] [Q&A]
Hi, I'm currently doing a course where Chakra-ui is present, and we're using the system present in chakra ui to style with css our components,  the problem seems that all the style props like  `color='gray.500'`   or `fontSize='lg'` aren't working properly. Browser console reports the next behavior:        
![image](https://user-images.githubusercontent.com/92234457/184182233-620a54eb-dcc2-4918-a883-ed0fb37c5812.png)

the thing is some properties are actually working like the `fontWeight='bold'` property.

Is there anyway to fix this issue or anyone knows what's going on or what might be causing this?

Code Below:
![image](https://user-images.githubusercontent.com/92234457/184183437-a6b1dc93-fe56-4c86-8488-176bc8fd573d.png)

and also chakra version & dependencies below:
![image](https://user-images.githubusercontent.com/92234457/184183554-7adf2ccf-d9e0-4df1-a133-5803029969f0.png)

thanks for any help, it is really appreciated

**Comment by mohammadizanloo55** [comment]
please duplicate your problem in the [codesandbox](https://codesandbox.io) To we understand your problem and try to solve it
